components/pedigree/resources/src/main/resources/pedigree/view/personHoverbox.js

```diff
@@ -158,6 +176,82 @@ define([
         },
 
         /**
+         * Generates alive, alive and well and deceased radio button section
+         *
+         * @method generateAliveWell
+         * @return {Raphael.el} Raphael element set
+         */
+        generateAliveWell: function() {
+            var node = this.getNode();
+            var lifeStatus = node.getLifeStatus();
+            var aliveandwellStatus = node.getAliveAndWell();
+            //generate 3 radio buttons
+            var labeles = ["Alive", "Alive & Well", "Deceased"];
+            var ticks = [];
+            var rects = [];
+            var height = this.getY()+this._height-this.getLabelsOffset()-10;
+            var aliveAndWell = editor.getPaper().set();
+            for (var i = 0; i < 3; i++) {
+                var circle = this.generateCircle(this.getX()+15, height+20*i, false);
+                var tick = this.generateCircle(this.getX()+15, height+20*i, true);
+                tick.hide();
+                if ((lifeStatus == "alive" && !aliveandwellStatus && i == 0)
+                    || (aliveandwellStatus && i == 1)
+                    || (lifeStatus == "deceased" && i == 2) ) {
+                    tick.show();
+                    tick.radioSelected = true;
+                }
+                ticks[i] = tick;
+
+                var text = editor.getPaper().text(this.getX()+25, height+20*i, labeles[i]).attr(PedigreeEditorParameters.attributes.awLabel);
+                var rect = editor.getPaper().rect(this.getX()+5, height+20*i-10, this._width-20, 20, 1).attr(PedigreeEditorParameters.attributes.awRect);
+                rect.tick = tick;
+                rects[i] = rect;
+                if (this._hidden && !this.isMenuToggled()) {
+                    circle.hide();
```

Comment:
actually, the fastest way is to have just one tick, and hide/show as usual, and on tick just move position. Less drawing, less hassle. Bsically you wont even need special hide/show code at all, it will hide and show together with everything. And inlyon state change code will move the position - even if it is hidden. And once shown it will show and in the correct place, and nothign special to hide or show